cmd/minishift/cmd/start_preflight.go

```diff
@@ -364,12 +370,38 @@ func checkLibvirtDefaultNetworkActive() bool {
 
 // checkHypervDriverSwitch returns true if Virtual Switch has been selected
 func checkHypervDriverSwitch() bool {
-	switchEnv := os.Getenv("HYPERV_VIRTUAL_SWITCH")
-	if switchEnv == "" {
+	switchEnvName := "HYPERV_VIRTUAL_SWITCH"
+	posh := powershell.New()
+	defer posh.Close()
+
+	switchEnvValue := os.Getenv(switchEnvName)
+
+	if switchEnvValue == "" {
+		checkIfDefaultSwitchExists := `Get-VMSwitch -Id c08cb7b8-9b3c-408e-8e30-5e16a3aeb444 | ForEach-Object { $_.SwitchType }`
```

Comment:
I think comment is needed here.
What is `Id` indicate here? Is it fixed? Will it be same for every user?

Reply:
Ah it is being called the Magic ID :)

Reply:
It is a static ID that the identity of the Default Switch. it will always be the same, yes.